tests/app/parser/test_metadata_parser.py

```diff
@@ -308,6 +308,26 @@ def test_invalid_schema_metadata(self):
 
         self.assertEqual('Invalid validator for schema metadata - invalidValidator', ite.exception.args[0])
 
+    def test_optional_metadata(self):
+        metadata = {
```

Comment:
`claims` or `metadata`? 😂

Reply:
🤷‍♂️

Reply:
You don't need this test...
As the behaviour is already checked as long as you add an optional metadata into `self`.